freight/notifiers/generic.py

```diff
@@ -0,0 +1,46 @@
+__all__ = ["GenericNotifier"]
+
+from freight import http
+from freight.models import App, Task, TaskStatus, User
+
+from .base import Notifier, NotifierEvent, generate_event_title
+
+
+class GenericNotifier(Notifier):
```

Comment:
Is there a better name for this? Http maybe?

Reply:
WebhookNotifier?